driver/src/test/scala/za/co/absa/hyperdrive/driver/TestSparkIngestor.scala

```diff
@@ -41,6 +40,7 @@ class TestSparkIngestor extends FlatSpec with BeforeAndAfterEach with MockitoSug
   private val configuration = new BaseConfiguration
 
   override def beforeEach(): Unit = {
+    spark.sparkContext
```

Comment:
Please add a comment that you are doing this in order to initialize the spark session (if that's the reason)